buildout.cfg

```diff
@@ -11,7 +11,6 @@ RestrictedPython =
 
 [interpreter]
 recipe = zc.recipe.egg
-interpreter = python
```

Comment:
What does this change have to do with the pull request?

Reply:
This causes problem if the buildout is run inside a virtualenv.